### ISSUE TITLE
Exporting: Object mask is not respected

### ISSUE THREAD

**Issue by Daerst** [armory3d/armorpaint#151]
Armor Paint v0.6 (2019-06-27) - ea12196 on Windows 10

1. Import an FBX with two different objects (e.g. GunMetal, GunGrip).
1. Import some materials.
1. Drop one material into the Layers list (making it a Fill Layer), and limit it to GunMetal.
1. Drop another material into the Layers list (making it a Fill Layer), and limit it to GunGrip.
1. Export as PNG or JPG.
--> The materials are not exported correctly, looks just like plain color to me.

**Comment by Daerst** [comment]
Seems this problem also holds for paint layers (at least the ones I converted using 'To Paint Layer'), so I changed the title.

Currently, this prevents me from painting on a single sub-object of my imported geometry and together with #153, my only chance is to individually export the parts as FBXs, paint them individually and combine the exported maps lateron. Any improvement on this workflow would be a huge timesaver.
